docs/semgrep-app/findings.md

```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
 
 <RemoveRule />
 
-#### Viewing and adding comments to finding's history
+#### Viewing and adding comments to finding's activity
```

Comment:
We are now adding notes, not comments as it seems: <img width="371" alt="Screenshot 2023-01-31 at 21 51 47" src="https://user-images.githubusercontent.com/26669307/215819121-2304a2b1-9e05-4960-a79e-692d38ae5449.png">

```suggestion
#### Viewing and adding notes to findings
```